test/components/project/show.spec.js

```diff
@@ -166,19 +166,9 @@ describe('ProjectShow', () => {
         });
         const li = app.findAll('#page-head-tabs li');
         const text = li.map(wrapper => wrapper.get('a').text());
-        text.should.eql(['Overview', 'Datasets']);
+        text.should.eql(['Overview', 'Datasets New']);
         li[0].should.be.visible(true);
       });
-
-      it('does not show tabs if project does not have a dataset', async () => {
```

Comment:
Could you also update the title of the previous test (removing "if the project has a dataset")?